packages/core/package.json

```diff
@@ -4,7 +4,8 @@
   "description": "UMA smart contracts and unit tests",
   "dependencies": {
     "@truffle/contract": "^4.2.20",
-    "@uma/common": "^1.0.1"
+    "@uma/common": "^1.0.1",
+    "buidler-gas-reporter": "^0.1.4"
```

Comment:
nit: should this be a dev dependency since packages that import this package don't need it?

Reply:
ye, you right & done :)